examples/storybook/src/shared/utils/fileUploadUtil.ts

```diff
@@ -3,8 +3,9 @@ import type {
   ImageComponentData,
   UpdateEntityFunc,
   VideoComponentData,
+  AudioComponentData,
 } from 'wix-rich-content-common';
-import { testImages, testWixVideos } from './mock';
+import { testImages, testWixVideos, testWixAudios } from './mock';
```

Comment:
```suggestion
import { testImages, testWixVideos, testAudios } from './mock';
```

Change the name to audio - this is not something unique to wix only